src/contracts/atlas/AtlasFactory.sol

```diff
@@ -133,7 +131,7 @@ contract AtlasFactory {
                 executionEnvironment := create2(0, add(creationCode, 32), mload(creationCode), memSalt)
             }
 
-            emit NewExecutionEnvironment(executionEnvironment, user, dAppControl, callConfig);
+            // emit NewExecutionEnvironment(executionEnvironment, user, dAppControl, callConfig);
```

Comment:
Why commenting out the event?

Reply:
The thought was just to save gas.  My thinking is that the solvers will have already seen the UserOp anyway and the EE gets created no matter what, so I don't think there's much value in the event itself - it isn't what solvers would listen for i don't think.